inc/breadcrumb.php

```diff
@@ -641,7 +641,7 @@ protected function add_term_archive_items() {
 		if ( false !== $taxonomy->rewrite ) {
 
 			// If 'with_front' is true, dd $wp_rewrite->front to the trail.
-			if ( $taxonomy->rewrite['with_front'] && $wp_rewrite->front )
+			if ( array_key_exists('with_front', $taxonomy->rewrite) && $taxonomy->rewrite['with_front'] && $wp_rewrite->front )
```

Comment:
Se deve essere controllata l'esistenza, perché non semplicemente `! empty()`?

Reply:
ci potrebbero essere dei casi in cui with_font non era valorizzata, ecco perche questo check in più

Reply:
Sì; intendevo che `! empty($taxonomy->rewrite['with_front'])` ha lo stesso effetto ed è più leggibile e conciso.